hooks/playbooks/ceph-bm.yml

```diff
@@ -0,0 +1,39 @@
+- name: Deploy ceph on OSP BMO deployed nodes
+  hosts: "{{ cifmw_target_hook_host | default('localhost') }}"
+  gather_facts: false
+  tasks:
+    - name: Fetch OSP BMO nodesets
+      environment:
+        KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+        PATH: "{{ cifmw_path }}"
+      ansible.builtin.command:
+        cmd: >-
+          oc get OpenStackBaremetalSet -n openstack -o yaml
```

Comment:
is the $namespace something we should pass as a parameter? I saw this pattern somewhere and I'm wondering if it makes sense to have a `$namespace | default('openstack')`.

Reply:
`namespace` is passed down by the run_hook role:
https://github.com/openstack-k8s-operators/ci-framework/blob/0f9117448b23647fa9051064c3e23cad87404651/roles/run_hook/tasks/playbook.yml#L35
You can/should consume it.

Reply:
It is possible but I don't think it is actually required having that I don't think anyone is using a different namespace for openstack other than openstack but lets hear some more opinions